python/ray/gcs_utils.py

```diff
@@ -55,6 +55,9 @@
 # Actor pub/sub updates
 RAY_ACTOR_PUBSUB_PATTERN = "ACTOR:*".encode("ascii")
 
+# Reporter pub/sub updates
+RAY_REPORTER_PUBSUB_PATTERN = "RAY_REPORTER.*".encode("ascii")
```

Comment:
Note the pubsub pattern is done with `.` (period) instead of `:` (colon) unlike actor pubsub channel. We should make this consistent.